dbt/include/teradata/macros/show.sql

```diff
@@ -1,7 +1,7 @@
--- This macro is overriden because limit keyword doesnot work in teradata database and has been replaced with top keyword
-{% macro teradata__get_limit_subquery_sql(sql, limit) %}
-    select top {{ limit }} *
-    from (
-        {{ sql }}
-    ) as model_limit_subq
+-- This macro is overriden because limit keyword doesnot work in teradata database and has been replaced with sample keyword
+{% macro teradata__get_limit_sql(sql, limit) %}
+   {{ compiled_code }}
```

Comment:
What is compiled_code here?

Reply:
compiled code is the sql only
https://github.com/dbt-labs/dbt-adapters/blob/f1987d4313cc94bac9906963dff1337ee0bffbc6/dbt/include/global_project/macros/adapters/show.sql#L22
It is coming form the upstream macro